docs/commands.md

```diff
@@ -498,6 +499,41 @@ redis> RG.PYSTATS
 6) (integer) 5745816
 ```
 
+## RG.PYDUMPREQS
+The **RG.PYDUMPREQS** command returns a list of all the python requirements available (with information about each requirement).
+
+**Redis API**
+
+```
+RG.PYDUMPREQS
+```
+
+_Return_
+
+An array that consists of alternating key name and value entries as follows:
+
+* **GearReqVersion**: an internal version of the requirement given by redis gears (do not confuse between this and the requirement python version)
+* **Name**: the name of the requirement as it was given to the REQUIREMENTS argument on [RG.PYEXECUTE](commands.md#rgpyexecute) command
+* **IsDownloaded**: true if the requirement wheels was successfully download otherwise false
```

Comment:
```suggestion
* **IsDownloaded**: `#!python True` if the requirement's wheels were successfully downloaded, `#!python False` otherwise
```

Reply:
Its not python True, just the text 'true', maybe I should change it to yes/no?

Reply:
Oh, right.

For RESP2 I'd stick with yes/no or 1/0. RESP3 has a Boolean type so that's a no-brainer.